gnomad/resources/grch37/gnomad.py

```diff
@@ -212,5 +212,9 @@ def release_vcf_path(data_type: str, version: str, contig: str) -> str:
     :param contig: Single contig "1" to "Y"
     :return: Path to VCF
     """
-    contig = f".{contig}" if contig else ""
```

Comment:
We still need this line right?

Reply:
Oops. Yes we do.

Reply:
Restored in f3eaf303c222d4e74069640c79cd8cc36ef6bdc1.